test/e2e/integreatly_test.go

```diff
@@ -62,9 +64,42 @@ func TestIntegreatly(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to add custom resource scheme to framework: %v", err)
 	}
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup()
+	err = ctx.InitializeClusterResources(&framework.CleanupOptions{TestContext: ctx, Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval})
+	if err != nil {
+		t.Fatalf("failed to initialize cluster resources: %v", err)
+	}
+	t.Log("Initialized cluster resources")
+	if err != nil {
+		t.Fatal(err)
+	}
+	// get global framework variables
+	f := framework.Global
+
+	apiextensions, err := clientset.NewForConfig(f.KubeConfig)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	testingContext := &common.TestingContext{
+		Client:     f.Client.Client,
+		KubeConfig: f.KubeConfig,
+		KubeClient: f.KubeClient,
+		ExtensionClient: apiextensions,
+	}
+
 	// run subtests
 	t.Run("integreatly", func(t *testing.T) {
-		t.Run("Cluster", IntegreatlyCluster)
+		for _, test := range common.ALL_TESTS {
+			t.Run(test.Description, func(t *testing.T) {
+				test.Test(t, testingContext)
+			})
+		}
+
+		t.Run("Cluster", func(t *testing.T) {
```

Comment:
This is passing off to the old not-yet-refactored tests, is that right?

Reply:
Yes, that's right.